CHANGELOG.md

```diff
@@ -1,41 +1,55 @@
-- Added `patch` method to the `Fork` structure. (#393)
-- Added a public `helthcheck` endpoint. (#405)
-- Added serialization support floating point types through special wrapper(`F32` and `F64`). This feature is hidden behind `float_serialize` gate. Note: special values (Infinity and NaN) aren't supported. (#384)
-
-### Changed
+## 0.5 - 2018-01-29
+
+### Breaking changes
+- The order of bytes and bits in the `DBKey` keys of `ProofMapIndex` became consistent. The change influences how
+  Merkle Patricia trees are built for `ProofMapIndex`: the bits in each byte of a `DBKey` are now enumerated from the
+  least significant bit (LSB) to the most significant bit (MSB), compared to MSB-to-LSB ordering used before. Note:
+  this change will break old storages using map proofs. (#419)
+- The `Database` trait is simplified: it is no longer required to implement state-sharing `clone` method.
+  Instead, the `merge` method now takes a shared reference to `self`. (#422)
+- `message!` and `encoding_struct!` no longer require manual `SIZE` and offset specification. (#413)
+- `from_raw(raw: RawMessage)`  method is moved to the `Message` trait. To migrate, add `use exonum::messages::Message`.
+  (#427)
 - Changed iterators over `Patch` and `Changes` data into custom types instead of standard collection iterators. (#393)
-- Fixed typo in `SparceListIndexKeys` and `SparceListIndexValues` (#398)
-- Fixed #15 consensus on the threshold of 1/3 sleeping validators. (#388)
+- Fixed typo in `SparceListIndexKeys` and `SparceListIndexValues`. (#398)
+- Removed default `state_hash` implementation in the `Service` trait. (#399)
+- Removed `info` method from the `Transaction`. (#402)
 - Replaced config param `timeout_events_capacity` with `internal_events_capacity`. (#388)
-- The `Transaction` trait now inherit `ExonumJson`. (#402)
-- The list of peer connections is now restored to the last state after the process is restarted. (#378)
-- `message!` and `encoding_struct!` no longer require manual `SIZE` and offset specification.
-- `from_raw(raw: RawMessage)` method is not part of the `Message` trait, and not an inherent method. (#427)
-- Log dependency was updated to 0.4, which can cause issues with previous versions. (#433)
-- The `Database` trait is simplified: it is not longer required to implement state-sharing `clone` method.
-  Instead, the `merge` method now takes a shared reference to `self`.
-- The order of bytes and bits in the `DBKey` keys of `ProofMapIndex` became consistent. The change influences how Merkle Patricia trees are built for `ProofMapIndex`: the bits in each byte of a `DBKey` are now enumerated from the least significant bit (LSB) to the most significant bit (MSB), compared to MSB-to-LSB ordering used before. Note: this change will break old storages using map proofs. (#419)
+- The `Transaction` trait now inherits from `ExonumJson`. (#402)
 - `DBKey` renamed to `ProofPath` and part of its functionality was moved to the `BitsRange` trait. (#420)
```

Comment:
Let's paraphrase this sentence in conformity with the sounding of the rest of the doc:
"Renamed `DBKey` to `ProofPath` and moved a part of its functionality to the `BitsRange` trait. (#420)"